packages/xen-api/src/index.js

```diff
@@ -1221,7 +1101,7 @@ Xapi.prototype._transportCall = reduce(
         .call(this._call(method, args), HTTP_TIMEOUT)
         .catch(error => {
           if (!(error instanceof Error)) {
-            error = wrapError(error)
+            error = XapiError.wrap(error)
```

Comment:
You could keep using `wrapError`?

Reply:
I think it makes more use as a static method on `XapiError`.